modules/installation-guide/partials/proc_importing-external-or-additional-tls-certificates.adoc

```diff
@@ -0,0 +1,218 @@
+
+
+[id="importing-external-or-additional-tls-certificates_{context}"]
+= Importing external or additional TLS certificates to {prod-short}
+
+Network communications between the components of a {prod-short} installation and the started workspaces, are all secured through the TLS protocol, and thus require the use of trusted certificate authorities. 
+
+In some cases it can be necessary to add TLS certificates to the {prod-short} installation, so that every {prod-short} component will consider them as signed by a trusted CA.
+
+Typical cases that may require this addition are:
+
+* when the underlying Kubernetes cluster uses TLS certificates signed by a CA that is not trusted,
+* when {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
+
+To store those certificates, {prod-short} uses a dedicated ConfigMap. Its default name is `ca-certs` but {prod-short} allows configuring its name.
+On OpenShift, the {prod-short} operator even injects cluster trusted certificates into this ConfigMap automatically.
```

Comment:
IMHO the OpenShift case should be a `[NOTE]` and should provide more details:
- link to latest documentation that explains how the certs are propagated
- the labels/annotation that the operator applies so that the propagation happen
Also consider that when this documentation is downstreamed OpenShift is the only platform. This means that a sentence like "On OpenShift..." doesn't make sense because it's always on OpenShift.

Reply:
What do you think of this:
![image](https://user-images.githubusercontent.com/686586/97467202-3e61cf00-1944-11eb-8983-76d7f9d01c98.png)

The link you see in this screenshot points to  https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki

Reply:
Fixed in commit https://github.com/eclipse/che-docs/pull/1598/commits/3f481ae07d585dc12a78fe6300ba5daea7659c93